src/web/app/pages-instructor/instructor-home-page/instructor-home-page.component.ts

```diff
@@ -159,22 +160,23 @@ export class InstructorHomePageComponent extends InstructorSessionBasePageCompon
    * Loads the feedback session in the course.
    */
   loadFeedbackSessions(model: CourseTabModel): void {
-    this.httpRequestService.get('/sessions', {
-      courseid: model.course.courseId,
-    }).subscribe((response: FeedbackSessions) => {
-      response.feedbackSessions.forEach((feedbackSession: FeedbackSession) => {
-        const m: SessionsTableRowModel = {
-          feedbackSession,
-          responseRate: '',
-          isLoadingResponseRate: false,
-          instructorPrivilege: defaultInstructorPrivilege,
-        };
-        model.sessionsTableRowModels.push(m);
-        this.updateInstructorPrivilege(m);
-      });
-
-      model.isTabExpanded = true;
-    }, (resp: ErrorMessageOutput) => { this.statusMessageService.showErrorMessage(resp.error.message); });
+    if (model.sessionsTableRowModels.length === 0) {
```

Comment:
This does not necessarily mean the data has not been loaded before. It could very well be the case that the course has no sessions.

Reply:
I agree. Any suggestion on this? 
```sessionsTableRowModels: SessionsTableRowModel[]``` and cannot be assigned `null` or `undefined`. Although an empty array bypasses this check, the current implementation allows the tab to expand and unexpand correctly.

Reply:
The data model could have a field value e.g. `hasPopulated` which will be set only when data has been fetched.